plugins/module_utils/podman/podman_container_lib.py

```diff
@@ -967,6 +967,7 @@ def diffparam_log_driver(self):
         return self._diff_update_and_compare('log_driver', before, after)
 
     def diffparam_log_level(self):
+        after = before = self.module.params.get('log_level')
```

Comment:
Shouldn't we stick to `after = self.params['log_level']` because `self.params` is filled with defaults while `self.module.params` is not?

Reply:
For some modules (e.g. `podman_containers`) `self.module_params` and `self.module.params` can be different, so maybe `self.module_params` is what we want here?